webapp-django/crashstats/crashstats/decorators.py

```diff
@@ -65,3 +68,21 @@ def inner(request, *args, **kwargs):
             raise
         return view(request, *args, **kwargs)
     return inner
+
+
+def track_api_pageview_decorator(view):
```

Comment:
I think I'll go for `from crashstats.base.ga import track_api_pageview as _track_api_pageview` or something such.